ros2_ws/src/warehouse_robot/config/my_controllers.yaml

```diff
@@ -9,6 +9,20 @@ controller_manager:
     joint_broad:
       type: joint_state_broadcaster/JointStateBroadcaster
 
+    # joint_piston:
+      # type: position_controllers/JointGroupPositionController
+
+    joint_piston:
+      type: forward_command_controller/ForwardCommandController
+
+  # joint1_position_controller:
+  #   type: effort_controllers/JointPositionController
+  #   joint: joint_piston
+  #   pid: {p: 1000, i: 0, d: 0}
```

Comment:
why are these and some of the lines above commented out?  should they just be removed instead?  If not, a quick comment detailing why they're commented out but not removed would be valuable.

Reply:
That was the original from the `warehouse_robot` and I tried to closely match it. It's not needed here though, I'll remove it